### PR TITLE
Updates for TP table based on current knowledge and formatting

### PR DESCRIPTION
**Pull request by codyhoag**
FYI @vikram-redhat

### CODE REVIEW

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -797,21 +798,21 @@ In the table below, features are marked with the following statuses:
...
+|DEP
+|DEP
+|DEP
 
 |OpenShift Ansible Service Broker
-|-
-|-
+|DEP
+|DEP
 |-
```
Comment (codyhoag) [inline release_notes/ocp-4-4-release-notes.adoc:817]: @adellape did I get the Service Catalog, TSB, and ASB statuses right in the TP table? I based those statuses off what I saw in the tracker comments and what we currently have in the RNs.

Comment (codyhoag) [reply release_notes/ocp-4-4-release-notes.adoc:817]: Confirmed via Slack :+1:

**Comment by codyhoag** [comment]
Most of these have been confirmed. The `Central Audit` feature I'm still waiting to hear back for (whether deprecated/removed for 4.2-4.4). I'll add any follow up tweaks in another PR.

**Comment by bobfuru** [comment]
LGTM

**Comment by vikram-redhat** [comment]
@codyhoag thanks! Add the updated link to team-docs too if not already for the team to see.

**Pull request merged**